.ci/filter_docker_aarch64.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+GOPATH_LOCAL="${GOPATH%%:*}"
+kata_dir="${GOPATH_LOCAL}/src/github.com/kata-containers"
+test_dir="${kata_dir}/tests"
+ci_dir="${test_dir}/.ci"
+test_config_file="${ci_dir}/configuration_aarch64.yaml"
+
+describe_filter_flag="docker.Describe"
+context_filter_flag="docker.Context"
+it_filter_flag="docker.It"
+
+# value for '-focus' in ginkgo
+_focus_options=()
+
+filter_and_build()
+{
+	dependency="$1"
+	array_docker=$("${GOPATH_LOCAL}/bin/yq" read "${test_config_file}" "${dependency}")
+	[ "${array_docker}" = "null" ] && array_docker="" && return
```

Comment:
I don't understand the assignment here. Can't it just be the following?

```
[ "${array_docker}" = "null" ] && return
```

Same comment applies to the file below.

Reply:
yes, the assignment of "" would be redundant. updated asap.